### ISSUE TITLE
Terraform scratch file

### ISSUE THREAD

**Issue by nicojaraiz** [ciberado/automation-workshop#2]
What would be the command to be able to view the draft created before using the "terraform apply" when finishing a "plan"

![Captura](https://github.com/user-attachments/assets/f05ee6c0-1127-4466-86af-2cfe2adba6e3)

**Comment by ciberado** [comment]
Here it is the relevant doc! https://developer.hashicorp.com/terraform/cli/commands/plan#out-filename. I will keep this issue opened until I find the time to update the lab :) Thanks!

_[-out=FILENAME](https://developer.hashicorp.com/terraform/cli/commands/plan#out-filename) - Writes the generated plan to the given filename in an opaque file format that you can later pass to terraform apply to execute the planned changes, and to some other Terraform commands that can work with saved plan files_

**Comment by nicojaraiz** [comment]
Thanks for the comment Ciberado!
I find it interesting to say the least, to be clear, before performing the "apply" I enter the command attached in your comment so I can preview what is configured, right?

**Comment by ciberado** [comment]
Yes :) Plan will always help you to preview what's going to happen. But if you use `out` the changelog will be stored in a file, so it can be directly applied later without recalculating anything. It is a way of ensuring that what you checked in the `plan` will be the actual mutation run by `apply`.
